dizoo/atari/envs/atari_env.py

```diff
@@ -93,6 +93,35 @@
         ),
         use_wrappers=None,
     ),
+    'EnduroNoFrameskip-v4': BaseEnvInfo(
+        agent_num=1,
+        obs_space=EnvElementInfo(
+            shape=(210, 160, 3),
+            value={
+                'min': 0,
+                'max': 255,
+                'dtype': np.float32
+            },
+        ),
+        act_space=EnvElementInfo(
+            shape=(9, ),
```

Comment:
For discrete action space, we should set shape to `(1, )`, size 1 with 9 discrete choices. Previous random_collect_size bug is related to it.